### DISCUSSION TITLE
could you please share any page giving details on throughput tests done with loxilb ?

### DISCUSSION THREAD

**Discussion by megarajan** [loxilb-io/loxilb#902] [General]
could you please share any page giving details on throughput tests done with loxilb ?

**Comment by TrekkieCoder** [comment]
> could you please share any page giving details on throughput tests done with loxilb ?

You can find the details in these links:
1. Non-kubernetes (bare-metal standalone container) [report](https://github.com/loxilb-io/loxilbdocs/blob/main/docs/perf.md)
2. Kubernetes performance [details](https://dev.to/nikhilmalik/l4-l7-performance-comparing-loxilb-metallb-nginx-haproxy-1eh0)
3. Performance on AWS EKS [here](https://www.loxilb.io/post/build-a-high-performance-eks-cluster-using-auto-scaled-loxilb)
4. RabbitMQ performance with loxilb service-proxy mode [here](https://www.loxilb.io/post/loxilb-cluster-networking-elevating-k8s-networking-capabilities)
5. Publication related to performance with [loxilb](https://arxiv.org/pdf/2405.00078) and eBPF security.
6. Performance on [ARM arch](https://www.loxilb.io/post/running-loxilb-on-aws-graviton2-based-ec2-instance)
7. Demo [video](https://www.youtube.com/watch?v=MJXcM0x6IeQ) on loxilb performance

**Comment by megarajan** [comment]
ok thanks will have a look at these ...
